Hazel/src/Hazel/Core/Core.h

```diff
@@ -43,22 +43,6 @@
 	#error "Unknown platform!"
 #endif // End of platform detection
 
-
-// DLL support
-#ifdef HZ_PLATFORM_WINDOWS
```

Comment:
Does Cherno really want go static-lib with his game engine?

Reply:
Yes, since he converted to the static lib, he sometimes removes a HAZEL_API when he comes across it, and in the video about converting to a static lib he mentioned this should be removed in a maintenance.